python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -81,20 +82,22 @@ def manifest_path(self) -> Path:
         return self._manifest_path
 
     @public
-    def prepare(self, *, quiet=False) -> None:
+    def prepare_for_deployment(self, *, log_level=logging.INFO) -> None:
         """A method that can be called as part of the deployment process to handle
-        preparing the manifest and if packaged_project_dir is set, handle copying
-        the dbt project to that directory.
+        all potential preparation steps.
```

Comment:
Use of "potential" is a little weird here. I also think calling out the steps specifically would make this less scary.